app/tournament/match.js

```diff
@@ -0,0 +1,39 @@
+export class Match {
+    constructor(round) {
+        this.round = round;
+        this.players = [];
+        this.isFinished = false;
+    }
+
+    addPlayer(player) {
+        if (this.players.length >= 2) {
+            throw new Error("Match has already 2 players");
+        }
+        this.players.push(player);
+    }
+
+    setNextMatch(match) {
+        this.nextMatch = match;
+    }
+
+    setWinner(player) {
+        if (!this.isFinished) {
+            this.isFinished = true;
```

Comment:
This way you won´t be able to render the winner. Perhaps this should save the player and not just true.